### DISCUSSION TITLE
Realtime for specific row has suddenly stopped working

### DISCUSSION THREAD

**Discussion by juanantoniosantanamedina** [supabase/supabase#4300] [Questions]
Good evening,

I have notice that Realtime for specific rows has stopped working. Listener works fine for the whole table but when listening to a specific row the table event is not triggered.
Nothing has changed in the code, is there anything new that we must change?

My listener is like this:
```javascript
console.log('Listening user',userId);
this.supabaseClient
	.from<Record>(`suppliers:id=eq.${userId}`) 
	.on('*', (payload) => {
		console.log('***********');
		this.onTableEvent(payload);
	})
	.subscribe();
````

If I remove the row filter the listener works fine. Has the syntax changed for row listeners?

By the way, userId is a string, the field in the database is a uuid

Thanks in advance

**Comment by GaryAustin1** [comment]
There are/were some new issues this week with realtime.
https://github.com/supabase/supabase/issues/4201
Is a good summary.
Get lastest supabase.js 1.28.5 AND toggle all your replication switches on and off (insert,delete, etc.) are first things to try to get the current code.

**Comment by w3b6x9** [comment]
@juanantoniosantanamedina please see comment: https://github.com/supabase/supabase/issues/4201#issuecomment-986440945
